android/app/build.gradle

```diff
@@ -238,6 +238,14 @@ task copyDownloadableDepsToLibs(type: Copy) {
     into 'libs'
 }
 
+// Additional configuration
+project.ext.envConfigFiles = [
+    debug: ".env.staging",
```

Comment:
hmm,  I see you've added `.env.development` but here you have .staging. Does it work?

Reply:
A, nvm I see why it works, you did `ENVFILE=.env.development react-native run-android` in package.json